app/views/datasets/_contact.html.erb

```diff
@@ -1,8 +1,29 @@
 <%
+  default_name = "Contact #{@dataset.organisation.title} regarding this dataset"
+  link_attrs = {'data-ga-event': 'contact', 'data-ga-publisher': @dataset.organisation.name, 'class': 'govuk_link'}
+
+  if contact_name_for(dataset).present?
```

Comment:
I'm a ruby newbie so ignore me if I'm totally off base on this one, but I was thinking using a ternary operator might be more elegant here?
I.E something like
`email_name = contact_name_for(dataset).present? ? contact_name_for(dataset) : default_name`

I see why you would've gone with the if/else/end option for readability though